src/pages/Setting.tsx

```diff
@@ -0,0 +1,49 @@
+import ListHeader from '../components/list/ListHeader';
+import { authState } from '../store/recoil';
+import { useSetRecoilState } from 'recoil';
+import styled from 'styled-components';
+const Setting = () => {
+    const setAuth = useSetRecoilState(authState);
+    return (
+        <SettingContainer>
+            <ListHeader title={'더보기'} />
+            <ListContainer>
+                <SettingButton
```

Comment:
```suggestion
                <LogoutButton
```
네이밍이 직관적이지 않은것 같습니다.